cluster/gce/util.sh

```diff
@@ -2884,6 +2916,10 @@ function create-proxy-vm() {
       export PROXY_RESERVED_IP
       export PROXY_RESERVED_INTERNAL_IP
 
+      # pass back the proxy reserved IP
+      echo ${PROXY_RESERVED_IP} > ${KUBE_TEMP}/proxy-reserved-ip.txt
```

Comment:
this can be read back from kubeconfig.

Reply:
it can be retrieved from there. won't fix for now --- this approach has been used in different places as a typical convenient way to get subprocess returns in bash.